_unittest_ironpython/run_unittests.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import os
+import sys
+from datetime import datetime
+import time
+
+sys.path.append(r"C:\Program Files\AnsysEM\AnsysEM21.1\Win64\PythonFiles\DesktopPlugin")
+path_dir = os.path.join(os.path.abspath(os.path.dirname(__file__)), "..")
+sys.path.append(path_dir)
+
+os.environ["UNITTEST_CURRENT_TEST"] = "1"
+run_dir = os.path.abspath(os.path.dirname(__file__))
+print(run_dir)
```

Comment:
If we're going to use print statements within here (which I'm fine with here within a testing framework), let's at least make sure that we're printing out what we're printing out (i.e. `print('Working directory', run_dir)`).